### ISSUE TITLE
[2][Medium][App] Add PearAI to path for Windows

### ISSUE THREAD

**Issue by Fryingpannn** [trypear/pearai-app#326]

**Comment by Fryingpannn** [comment]
Being worked on by mimiq from Discord

**Comment by ICEPrey** [comment]
@MitchTM17 (mimiq)

**Comment by timknappe** [comment]
Was the CLI use already setup for Mac or is it somewhere in a PR? I'd need to take a look at the code

**Comment by MitchTM17** [comment]
As of now its (sort of) set up for Mac. It's not an option the user is prompted with on install or anything pretty, but you can add pear to the PATH like this, and then use `pearai .`
![image](https://github.com/user-attachments/assets/f027c2e0-afb6-4106-b45b-25c9bb1f4ce7)

Still working on actually finding the code where that's implemented though... if anyone has any ideas they would be greatly appreciated

**Comment by timknappe** [comment]
Yeah, thats my issue aswell, looked for the implementation but didnt find anything valuable. I saw some InnoScript files that were related to it, thats all.
